app/api/resources/user.py

```diff
@@ -276,11 +279,22 @@ def post(cls):
         if is_valid != {}:
             return is_valid, 400
 
-        result = DAO.create_user(data)
-
-        if result[1] is 200:
+        failed_send = False
+        try:
             send_email_verification_message(data["name"], data["email"])
 
+        except SMTPException as se:
+            failed_send = True
+            print("SMTP error: {0}".format(se))
+            return messages.SMTP_ERROR, 400
```

Comment:
Why is this considered bad request (400) and the other error is 5xx? Its really just a doubt of mine 🤔

Reply:
I changed this now to SMTPAuthenticationError and used HTTPStatus.UNAUTHORIZED which gives error code 401 on Swagger automatically.